test/unit/csv/invoices_exporter_test.rb

```diff
@@ -4,15 +4,15 @@ class Csv::InvoicesExporterTest < ActiveSupport::TestCase
 
   def setup
     @provider = FactoryBot.create(:simple_provider, org_name: 'FunkyTech', domain: 'funky-tech.com')
-    @buyer = FactoryBot.create(:simple_buyer, provider_account: @provider, org_legaladdress: 'Null street')
+    @buyer = FactoryBot.create(:old_simple_buyer, provider_account: @provider, org_legaladdress: 'Null street')
 
     @invoice_one = FactoryBot.create(:invoice,
                                       buyer_account: @buyer,
                                       provider_account: @provider,
                                       period: Month.new(Time.utc(2009, 6, 1)),
                                       created_at: Time.utc(2009, 6, 2))
```

Comment:
I think, in order to have more control over these tests, it would be better to create the old buyer explicitly (passing the date). Otherwise, anyone looking at it will wonder why `Time.zone.local(1984, 1, 1)` or `Time.utc(1964, 10, 1)`.
Why there should be an `old_buyer` factory but not an `old_invoice` once.

Reply:
Done! no `old_buyer`, all creation dates are specified explicitly, by travelling in time to create buyers.